node/index.js

```diff
@@ -614,7 +617,7 @@ TChannelConnection.prototype.checkOutOpsForTimeout = function checkOutOpsForTime
                 });
             continue;
         }
-        var timeout = op.options.timeout || self.channel.reqTimeoutDefault;
+        var timeout = op.req.ttl || self.channel.reqTimeoutDefault;
```

Comment:
are we populating the `ttl` from the `timeout` ?

Are `ttl` and `timeout` even the same thing ?

I think `ttl` is telling autobahn router "when should you stop retrying" where as `timeout` is telling a local tchannel client when you should you time out a pending operation.

Generally in production you want `ttl = timeout - fudgefactor` ?